src/store/pages.js

```diff
@@ -171,6 +171,14 @@ export default {
 			return state.pages.find(p => (p.parentId === parentId && p.title === TEMPLATE_PAGE))
 		},
 
+		pagePrintLink: (state, get) => (page) => {
+			const path = [page.collectivePath.split('/').at(-1), page.filePath.split('/'), page.fileName]
+			if (path.at(-1) === 'Readme.md') path.splice(-1, 1)
```

Comment:
We don't use these one-line `if () ...` syntax so far, I'd prefer to stick to if () { \n ... \n }` :grimacing: 

Same for the following two lines.